src/rx.cpp

```diff
@@ -81,10 +85,12 @@ Receiver::Receiver(const char *wlan, int wlan_idx, uint32_t channel_id, BaseAggr
         program = string_format("ether[0x0a:2]==0x5742 && ether[0x0c:4] == 0x%08x", channel_id);
 
         if (pcap_compile(ppcap, &bpfprogram, program.c_str(), 1, 0) == -1) {
+            WFB_LOG(LogLevel::ERROR, "Unable to compile %s: %s", program.c_str(), pcap_geterr(ppcap));
```

Comment:
@vertexodessa Why need to log before raising exception? It will print error twice: in a logger and in an exception handler

Reply:
good point, we don't need logging before exceptions, they can be ccaught on the calling side and logged.

Reply:
@svpcom fixed